### PR TITLE
refactor wizard and history view

### PR DESCRIPTION
**Pull request by Keith-CY**
1. clear data when switching between wizard subviews
2. add more response info in get-history API

### CODE REVIEW

File: packages/neuron-ui/src/components/History/index.tsx [hunk 1]
```diff
@@ -129,11 +132,13 @@ const History = (props: React.PropsWithoutRef<ContentProps & RouteComponentProps
...
       </ContextMenuZone>
-      <Row>
-        <Col>
-          <Pagination currentPage={pageNo} pageSize={pageSize} total={totalCount} onChange={onPageChange} />
-        </Col>
-      </Row>
+      {addresses.length > 0 ? (
+        <Row>
+          <Col>
+            <Pagination currentPage={pageNo - 1} pageSize={pageSize} total={totalCount} onChange={onPageChange} />
```
[the remark this answers is not in the document]
Comment (Keith-CY) [reply packages/neuron-ui/src/components/History/index.tsx]: That's the convention is the (component?) community cuz `pageNo` => `offset`, `pageSize` => `limit`

File: packages/neuron-ui/src/components/History/index.tsx [hunk 2]
```diff
@@ -82,20 +81,24 @@ const History = (props: React.PropsWithoutRef<ContentProps & RouteComponentProps
   const onPageChange = useOnChangePage(search, pathname, history, queryFormatter)
   const onAddressRemove = useOnAddressRemove(search, pathname, history, queryFormatter)
 
-  if (loadings.transactions) {
-    return <div>Loading</div>
-  }
+  // if (loadings.transactions) {
```
Comment (ashchan) [inline packages/neuron-ui/src/components/History/index.tsx]: Please don't leave comment as unused code. Either delete them, or finish them when possible.

File: packages/neuron-ui/src/components/History/index.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (ashchan) [reply packages/neuron-ui/src/components/History/index.tsx]: I have one question: for an end user, what does the link for first page show, `0` or `1`? It should be `1` regardless of param names we choose.

[the remark this answers is not in the document]
Comment (Keith-CY) [reply packages/neuron-ui/src/components/History/index.tsx]: pageNo starts from 1 which outsides the component, currentPage starts from 0 which insides the component. PageNo used in the link

File: packages/neuron-ui/src/components/History/index.tsx (hunk 2 above)
Comment (Keith-CY) [reply packages/neuron-ui/src/components/History/index.tsx]: removed in https://github.com/nervosnetwork/neuron/pull/312/commits/7f6980c25c9a7e80964a9a5f21bf3f417bdffc30

File: packages/neuron-ui/src/components/History/index.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (classicalliu) [reply packages/neuron-ui/src/components/History/index.tsx]: I still don't understand why `Pagination` component start with 0

[the remark this answers is not in the document]
Comment (Keith-CY) [reply packages/neuron-ui/src/components/History/index.tsx]: > I still don't understand why `Pagination` component start with 0

Just convention

**Pull request merged**